cnvlib/call.py

```diff
@@ -231,16 +216,18 @@ def _reference_expect_copies(chrom, ploidy, is_sample_female, is_reference_male)
         A pair of integers: number of copies in the reference, and expected in
         the sample.
     """
-    chrom = chrom.lower()
-    if chrom in ["chrx", "x"]:
-        ref_copies = (ploidy // 2 if is_reference_male else ploidy)
-        exp_copies = (ploidy if is_sample_female else ploidy // 2)
-    elif chrom in ["chry", "y"]:
-        ref_copies = ploidy // 2
-        exp_copies = (0 if is_sample_female else ploidy // 2)
-    else:
-        ref_copies = exp_copies = ploidy
-    return ref_copies, exp_copies
+    df = cnarr.copy().data
```

Comment:
Direct access of the `data` member. I initially had implemented a `CopyNumArray.get_copied_dataframe()` but realized that at several places throughout the code base `data` is accessed directly. One could replace all those with a dedicated method, though.